pkger/testdata/dashboard_gauge.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: 0.1.0
+kind: Package
+meta:
+  pkgName:      pkg_name
+  pkgVersion:   1
+  description:  pack description
+spec:
+  resources:
+    - kind: Dashboard
+      name: dash_1
+      description: desc1
+      charts:
+        - kind:   gauge
+          name:   gauge
+          note: gauge note
+          noteOnEmpty: true
+          xPos:  1
+          yPos:  2
+          width:  6
+          height: 3
+          shade: true
+          geom: line
+          legend:
```

Comment:
axe legend not applicable. Lets keep these files to only the vital data that is useful to a guage cahrt. axe the rest, is a bit confusing to see here and not above

Reply:
same here - copy-pasta

Reply:
right on, go on and axe the erroneous fields and we'll be ready to rock n roll here 👍